### ISSUE TITLE
Issues with car efficiency choice

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmobile#62]
The current text for fossilCarEfficiency reads:

"New cars become almost 2% more efficient every year. To reduce
      CO<sub>2</sub> emissions, it is important that people replace their
      old puffing machines with shiny new ones as technology improves.
    `"

I have two problems with this:
* To get a feeling for how much 2% per year amounts to, the user should know the end-year of the scenario. Is this clear in the game?
* This text suggests that people should buy more new cars. I'm not convinced that this would solve any environmental problems or reduce CO2.

Possible solutions:
* Remove the choice
* State the end-year (e.g., the number of years for which you apply this efficiency growth)
* Express the efficiency increase as a total % (instead of %/y)
* Remove the text about buying new cars

What do you think @dennisschoenmakers @antw?

**Comment by ChaelKruip** [comment]
For refinery efficiency, a similar issue occurs:
```
      Refineries use a lot of energy breaking down crude oil into oil products.
      In recent decades, these plants have become a lot more efficient. What
      if we keep making them 1% more efficient each year?
```

How many years do we do this?

**Comment by dennisschoenmakers** [comment]
> To get a feeling for how much 2% per year amounts to, the user should know the end-year of the scenario. Is this clear in the game?

No. @antw could you add a short introduction paragraph after people press "play the game" that reads: "It's 2050 now. What should you rather do as we need to reduce our CO2-emissions (close) to zero." Agree @ChaelKruip ?

**Comment by dennisschoenmakers** [comment]
@ChaelKruip could you update the text by including the total efficiency improvement?

**Comment by dennisschoenmakers** [comment]
Also read #87.
